src/store/initial-state.js

```diff
@@ -1,9 +1,11 @@
 import deepmerge from 'deepmerge';
-import { loadLocalStorage, saveLocalStorage } from './helpers';
+import { loadLocalStorage } from './helpers';
 import normalizeData from './normalize-data';
 import { getFlagsFromUrl, Flags } from '../utils/flags';
 import { settings, sidebarWidth, localStorageName, params } from '../config';
 
+const nodeTypes = ['parameters', 'task', 'data'];
```

Comment:
does this need to be global. Can it be local to the func.

Reply:
ah yeah i think we can have it inside a func 👍